### ISSUE TITLE
Transition property compiles to broken CSS with prefixing

### ISSUE THREAD

**Issue by gscottolson** [Khan/aphrodite#96]
```es
const styles = StyleSheet.create({
  transition: 'border-color 300ms ease-in-out',
})
```
produces:
```css
  transition: b-ms-order-color 300ms ease-in-out,b-webkit-order-color 300ms ease-in-out,border-color 300ms ease-in-out !important;
  -webkit-transition: b-webkit-order-color 300ms ease-in-out,border-color 300ms ease-in-out !important;
```

Notice that `border-color` has been mangled as `b-ms-order-color` and the like. I am also seeing `webkit-transition` styles with duplicate entries (and…mangled).

**Comment by xymostech** [comment]
Freaky! This looks like a bug in our auto-prefixer though, https://github.com/rofrischmann/inline-style-prefix-all I'll go report a bug there. :)

**Comment by xymostech** [comment]
Looks like there's already a bug report: https://github.com/rofrischmann/inline-style-prefix-all/issues/15

**Comment by ctrlplusb** [comment]
It's been fixed. :)

**Comment by jlfwong** [comment]
Would be great to have a test for this, along with a fix by upgrading `inline-style-prefix-all`! Would make a good first PR ;)

**Comment by gscottolson** [comment]
@jlfwong Thanks for the suggestion. PR is up.

**Comment by jlfwong** [comment]
Should be fixed in 0.4.1
